pom.xml

```diff
@@ -82,7 +83,7 @@
         <version.arquillian.cube>1.6.0</version.arquillian.cube>
         <version.eclipse.egit.github>2.1.5</version.eclipse.egit.github>
         <version.flapdoodle.embed.mongo>1.50.1</version.flapdoodle.embed.mongo>
-        <version.fusepatch>5.0.0</version.fusepatch>
+        <version.fusepatch>6.0.0-SNAPSHOT</version.fusepatch>
```

Comment:
What's the plan with the Galleon upgrade?

Are we just going to bump to WF 14 first and then work on the switch from Fuse Patch? If so, we'll need a new Fuse Patch release.

Reply:
Good point. Let me have a look at the commit from Thomas which moves some parts of fuse patch to wfc and removes the rest. Perhaps it can work even before Galleon.